python/ray/worker.py

```diff
@@ -1248,28 +1262,30 @@ def actor_handle_deserializer(serialized_obj):
     # We register this serializer on each worker instead of calling
     # register_custom_serializer from the driver so that isinstance still
     # works.
-    worker.serialization_context.register_type(
+    serialization_context.register_type(
         ray.actor.ActorHandle,
         "ray.ActorHandle",
         pickle=False,
         custom_serializer=actor_handle_serializer,
         custom_deserializer=actor_handle_deserializer)
 
+    worker.serialization_context_map[driver_hash] = serialization_context
+
     if worker.mode in [SCRIPT_MODE, SILENT_MODE]:
         # These should only be called on the driver because
         # register_custom_serializer will export the class to all of the
         # workers.
-        register_custom_serializer(RayTaskError, use_dict=True)
-        register_custom_serializer(RayGetError, use_dict=True)
-        register_custom_serializer(RayGetArgumentError, use_dict=True)
+        register_custom_serializer(RayTaskError, driver_hash, use_dict=True, local=True)
```

Comment:
Why `local=True`? When will these serializers get registered on the workers?

Reply:
I would prefer to register all the default types locally in order to reduce the load of redis, and it's more efficient to do in this way. Your comment reminds me the conditional statement outside this block, I will remove it. Thanks.